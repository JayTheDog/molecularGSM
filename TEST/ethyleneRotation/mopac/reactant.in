PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  C 0.665081 0.000000 0.000000
  C -0.664950 0.000000 0.000000
  H -1.238123 0.921902 -0.050699
  H 1.238057 -0.921995 0.050448
  H 1.238057 0.921995 -0.050448
  H -1.238123 -0.921902 0.050699
