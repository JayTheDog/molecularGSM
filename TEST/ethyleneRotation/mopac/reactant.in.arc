

                     SUMMARY OF PM6 CALCULATION, Site No: 14762

                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:52:07 2016
                                                       No. of days left =   0

           Empirical Formula: C2 H4  =     6 atoms

PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =         15.70744 KCAL/MOL =      65.71993 KJ/MOL
          TOTAL ENERGY            =       -298.56828 EV
          ELECTRONIC ENERGY       =       -747.59324 EV
          CORE-CORE REPULSION     =        449.02496 EV
          GRADIENT NORM           =          0.68570
          DIPOLE                  =          0.00012 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =          6
          IONIZATION POTENTIAL    =         10.676995 EV
          HOMO LUMO ENERGIES (EV) =        -10.677  1.160
          MOLECULAR WEIGHT        =         28.054
          COSMO AREA              =         71.33 SQUARE ANGSTROMS
          COSMO VOLUME            =         53.05 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     4    H     3     3.10002
            H     6    H     5     2.93442
            C     1    H     3     0.00001
          SCF CALCULATIONS        =          7
          WALL-CLOCK TIME         =          0.023 SECONDS
          COMPUTATION TIME        =          0.026 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  C     0.66348652 +1   0.00001902 +1   0.00000311 +1
  C    -0.66340720 +1   0.00001658 +1   0.00000263 +1
  H    -1.26046626 +1   0.90075146 +1  -0.04941787 +1
  H     1.26034691 +1  -0.90087790 +1   0.04940999 +1
  H     1.26048810 +1   0.90081326 +1  -0.04942025 +1
  H    -1.26032236 +1  -0.90080973 +1   0.04940831 +1
 
