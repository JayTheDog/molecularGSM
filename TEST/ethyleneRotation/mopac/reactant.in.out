 *******************************************************************************
 ** Site#: 14762        For non-commercial use only     Version 15.168L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.168L web: HTTP://OpenMOPAC.net         Days remaining:   0    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Dec  2 19:52:06 2016  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  NOSYM      - POINT-GROUP SYMMETRY SET TO C1
 *  T=         - A TIME OF     10.0 MINUTES REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.66508100  *   0.00000000  *   0.00000000  *
     2       C         -0.66495000  *   0.00000000  *   0.00000000  *
     3       H         -1.23812300  *   0.92190200  *  -0.05069900  *
     4       H          1.23805700  *  -0.92199500  *   0.05044800  *
     5       H          1.23805700  *   0.92199500  *  -0.05044800  *
     6       H         -1.23812300  *  -0.92190200  *   0.05069900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        0.6651    0.0000    0.0000
     2         C       -0.6650    0.0000    0.0000
     3         H       -1.2381    0.9219   -0.0507
     4         H        1.2381   -0.9220    0.0504
     5         H        1.2381    0.9220   -0.0504
     6         H       -1.2381   -0.9219    0.0507

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    6

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    12.595 HEAT:  16.03227
 CYCLE:     2 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    10.899 HEAT:  15.87974
 CYCLE:     3 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    19.232 HEAT:  15.85827
 CYCLE:     4 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     5.634 HEAT:  15.75033
 CYCLE:     5 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.653 HEAT:  15.71003
 CYCLE:     6 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     0.686 HEAT:  15.70744

     GRADIENT =  0.68570  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:52:07 2016
                                                       No. of days left =   0




          FINAL HEAT OF FORMATION =         15.70744 KCAL/MOL =      65.71993 KJ/MOL


          TOTAL ENERGY            =       -298.56828 EV
          ELECTRONIC ENERGY       =       -747.59324 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =        449.02496 EV
          COSMO AREA              =         71.33 SQUARE ANGSTROMS
          COSMO VOLUME            =         53.05 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.68570
          IONIZATION POTENTIAL    =         10.676995 EV
          HOMO LUMO ENERGIES (EV) =        -10.677  1.160
          NO. OF FILLED LEVELS    =          6
          MOLECULAR WEIGHT        =         28.054

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     4    H     3     3.10002
            H     6    H     5     2.93442
            C     1    H     3     0.00001


          SCF CALCULATIONS        =          7
 
          WALL-CLOCK TIME         =          0.023 SECONDS
          COMPUTATION TIME        =          0.025 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.66348652  *   0.00001902  *   0.00000311  *
     2       C         -0.66340720  *   0.00001658  *   0.00000263  *
     3       H         -1.26046626  *   0.90075146  *  -0.04941787  *
     4       H          1.26034691  *  -0.90087790  *   0.04940999  *
     5       H          1.26048810  *   0.90081326  *  -0.04942025  *
     6       H         -1.26032236  *  -0.90080973  *   0.04940831  *

                             CARTESIAN COORDINATES

   1    C        0.663486524     0.000019019     0.000003107
   2    C       -0.663407203     0.000016584     0.000002634
   3    H       -1.260466262     0.900751458    -0.049417870
   4    H        1.260346915    -0.900877896     0.049409989
   5    H        1.260488105     0.900813262    -0.049420250
   6    H       -1.260322358    -0.900809728     0.049408314


           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -26.86646 -20.59641 -16.01764 -14.46284 -12.04265 -10.67699   1.15990   5.45978
   5.74043   6.30432   6.49190   6.50566


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.286452        4.2865     1.11641     3.17004
    2          C          -0.286480        4.2865     1.11640     3.17008
    3          H           0.143234        0.8568     0.85677
    4          H           0.143232        0.8568     0.85677
    5          H           0.143238        0.8568     0.85676
    6          H           0.143227        0.8568     0.85677
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000    -0.000    -0.000     0.000
 HYBRID        -0.000    -0.000    -0.000     0.000
 SUM           -0.000    -0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.11641   1.03216   1.13748   1.00040
    2  C   1.11640   1.03216   1.13750   1.00042
    3  H   0.85677
    4  H   0.85677
    5  H   0.85676
    6  H   0.85677

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
